hazelcast-client/src/main/java/com/hazelcast/client/map/impl/nearcache/ClientHeapNearCache.java

```diff
@@ -57,70 +57,70 @@
+        this.executionService = (ClientExecutionServiceImpl) context.getExecutionService();
+        this.serializationService = context.getSerializationService();
+
+        this.inMemoryFormat = nearCacheConfig.getInMemoryFormat();
         if (inMemoryFormat != InMemoryFormat.BINARY && inMemoryFormat != InMemoryFormat.OBJECT) {
             throw new IllegalArgumentException("Illegal in-memory-format: " + inMemoryFormat);
         }
-        timeToLiveMillis = TimeUnit.SECONDS.toMillis(nearCacheConfig.getTimeToLiveSeconds());
-        invalidateOnChange = nearCacheConfig.isInvalidateOnChange();
-        evictionPolicy = EvictionPolicy.valueOf(nearCacheConfig.getEvictionPolicy());
-        selectedComparator = NearCacheRecord.getComparator(evictionPolicy);
-        cache = new ConcurrentHashMap<K, NearCacheRecord>();
-        canCleanUp = new AtomicBoolean(true);
-        canEvict = new AtomicBoolean(true);
-        lastCleanup = Clock.currentTimeMillis();
-        stats = new NearCacheStatsImpl();
+        this.maxSize = nearCacheConfig.getMaxSize();
+        this.maxIdleMillis = TimeUnit.SECONDS.toMillis(nearCacheConfig.getMaxIdleSeconds());
+        this.timeToLiveMillis = TimeUnit.SECONDS.toMillis(nearCacheConfig.getTimeToLiveSeconds());
+        this.invalidateOnChange = nearCacheConfig.isInvalidateOnChange();
+        this.evictionPolicy = EvictionPolicy.valueOf(nearCacheConfig.getEvictionPolicy());
+        this.canCleanUp = new AtomicBoolean(true);
+        this.canEvict = new AtomicBoolean(true);
+        this.stats = new NearCacheStatsImpl();
+        this.selectedComparator = NearCacheRecord.getComparator(evictionPolicy);
+        this.lastCleanup = Clock.currentTimeMillis();
     }
 
     @Override
-    public void put(K key, Object object) {
+    public void put(K key, Object value) {
         fireTtlCleanup();
-        if (evictionPolicy == EvictionPolicy.NONE && cache.size() >= maxSize) {
+        if (evictionPolicy == NONE && cache.size() >= maxSize) {
             return;
         }
-        if (evictionPolicy != EvictionPolicy.NONE && cache.size() >= maxSize) {
+        if (evictionPolicy != NONE && cache.size() >= maxSize) {
             fireEvictCache();
         }
-        Object value = null;
-        if (object != null) {
-            SerializationService serializationService = context.getSerializationService();
-            if (inMemoryFormat == InMemoryFormat.BINARY) {
-                value = serializationService.toData(object);
-            } else if (inMemoryFormat == InMemoryFormat.OBJECT) {
-                value = serializationService.toObject(object);
-            } else {
-                throw new IllegalArgumentException();
-            }
+        if (value == null) {
+            value = NULL_OBJECT;
+        } else if (inMemoryFormat == InMemoryFormat.OBJECT) {
+            value = serializationService.toObject(value);
+        } else {
+            value = serializationService.toData(value);
```

Comment:
I just picked the side I found easier to read. I can change it to the other style, but would stick with `value` as parameter name and `candidate` for the inline variable.